src/cogent3/parse/sequence.py

```diff
@@ -44,7 +44,7 @@ def FromFilenameParser(filename, format=None, **kw):
     - filename: name of the sequence alignment file
     - format: the multiple sequence file format
     """
-    format = format_from_filename(filename, format)
+    format, _ = format_from_filename(filename, format)
```

Comment:
the old function, use the new `get_format_suffixes()` and make sure it's imported